### PR TITLE
Added User Online Status

### LINKED ISSUE #48: Feat: Check if User is Online
**What is your issue related to ?**

- [x] Code
- [ ] Designing
- [ ] Documentation
- [ ] Testing/Research
- [ ] Others[please specify]

**Description**
Users can check if their Partners are online or not.

**Additional context**
Add after Implementation.

### PR DESCRIPTION
**Pull request by Rohit-2602**
## Description

Added User Online Status.

## Related Issue

Fixes #48  

## Type of Contribution:

- [ ] Bug fix
- [x] Coding
- [ ] Design
- [ ] Testing
- [ ] Research
- [x] Enhancement
- [ ] Documentation
- [ ] Other : 

## Checklist:

- [x] I have read README.md and CONTRIBUTION.md
- [x] My Pull request is made against `develop` branch.
- [x] My code/design follows the style/guidelines of this project.
- [x] I have performed a self-review of my own code/design/documentation.
- [ ] I have commented my code, particularly in hard-to-understand areas(or gave a proper description about the design).

### Screenshots (if applicable):

https://user-images.githubusercontent.com/65807152/122540541-607c9e80-d046-11eb-89c6-6938f7501926.mp4

### CODE REVIEW

**Comment by Rohit-2602** [comment]
@Acash512 Can you please review this PR.

**Comment by Rohit-2602** [comment]
@Acash512 changes done. Please Review.

File: Code/app/src/main/res/layout/list_item_partner.xml [hunk 1]
```diff
@@ -96,7 +101,7 @@
 
             </LinearLayout>
         </LinearLayout>
-    </LinearLayout>
+    </androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (Acash512) [inline Code/app/src/main/res/layout/list_item_partner.xml]: No use for constraint layout here, use linear layout as before, rest looks good

**Comment by Rohit-2602** [comment]
@Acash512 Done.

**Review by Acash512 — APPROVED** [review]
Great work @Rohit-2602. Thanks for your contribution

**Review by yatharthmago01 — APPROVED** [review]

**Review by SHITIZ-AGGARWAL — APPROVED** [review]

**Review by nidhisingh-1 — APPROVED** [review]

**Pull request merged**